kairon/shared/data/processor.py

```
@@ -254,6 +254,211 @@
+    def is_slot_in_database_action_config(config: dict, slot: Text):
+        payload = config.get('payload', [])
+        for item in payload:
+            if item.get('type') == 'from_slot' and item.get('value') == slot:
+                return True
+
+        set_slots = config.get('set_slots', [])
+        for set_slot in set_slots:
+            if set_slot.get('name') == slot:
+                return True
+
+        return False
```

Ruff

Use `return any(set_slot.get("name") == slot for set_slot in set_slots)` instead of `for` loop

Replace with `return any(set_slot.get("name") == slot for set_slot in set_slots)`

(SIM110)